blog-service/2023-08-09-metrics-slos.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Metrics-Based SLOs (Reliability Management)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - slo
+  - observability
+  - metrics
+  - operators
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+You can now create SLOs from the **Metrics** page and use [metrics operators](/docs/metrics/metrics-operators) in these SLO queries. The metrics query specified in your SLO should be a pure selector query. You can specify one or more operators in the query for SLO.
+
+[Learn more]
+
+NOTE: <!-- Does this mean not available to all customers yet? Or not all metrics operators available yet? -->
```

Comment:
This feature will be available to all customers. All metrics operators will be supported as well.